omc3/optics_measurements/phase.py

```diff
@@ -42,6 +50,15 @@
 
 LOGGER = logging_tools.get_logger(__name__)
 
+class CompensationMode:
+    NONE: str = "none"
+    MODEL: str = "model"
+    EQUATION: str = "equation"
+    
+    @classmethod
+    def all(cls) -> list[str]:
+        return [cls.NONE, cls.MODEL, cls.EQUATION]
```

Comment:
Do we want to make this an `Enum`? I'm not sure I understand well how they work though

Reply:
Not the biggest fan of enums (in pyhton). Also: As these are input-parameters, we don't want to confuse people with `--compensation 1` instead of `--compensation model` .